docs/diagrams/ArchitectureSequenceDiagram.puml

```diff
@@ -7,19 +7,19 @@ Participant ":Logic" as logic LOGIC_COLOR
 Participant ":Model" as model MODEL_COLOR
 Participant ":Storage" as storage STORAGE_COLOR
 
-user -[USER_COLOR]> ui : "delete 1"
+user -[USER_COLOR]> ui : "delete supplier 12"
 activate ui UI_COLOR
 
-ui -[UI_COLOR]> logic : execute("delete 1")
+ui -[UI_COLOR]> logic : execute("delete supplier 12")
```

Comment:
do remember to update the delete command here as well :)

Reply:
Yea this is outdated now, I have fixed it earlier. No wonder I couldn't find it hahaha

Reply:
Yes, sorry about that!